src/config.ts

```diff
@@ -128,6 +128,7 @@ export type ChainData = {
   socket: string
   contracts: Contracts
   betToken: BetToken
+  type: 'mainnet' | 'testnet'
```

Comment:
hm, maybe ok, but there is such a parameter in the chain entity:
`chainData.chain.testnet: boolean`

Reply:
yeah, it's a way better
f9d2241be2b8c653c06089935bb706f423f15450